src/docs/automation-and-resiliency/request-quota-increase-for-openshift-project-set.md

```diff
@@ -18,29 +18,37 @@ content_owner: Cailey Jones
 sort_order: 5
 ---
 # Request a quota increase for an OpenShift project set
```

Comment:
@w8896699 @Pilargit12 Jason Leach is working on a step-by-step document about quota increase request, could you check with him to avoid duplicates/conflicts?